core/cmd/local_client.go

```diff
@@ -160,10 +160,8 @@ func (cli *Client) runNode(c *clipkg.Context) error {
 			log.Fatal(err)
 		}
 	}()
-	err = logConfigVariables(lggr, cli.Config)
-	if err != nil {
-		return errors.Wrap(err, "failed to log config variables")
-	}
+
+	lggr.Debug("Environment variables\n", config.NewConfigPrinter(cli.Config))
```

Comment:
I wonder if it would be possible to also have it log structured data...

Reply:
Adding it would be easy, but changing to that format would be a bit messy since we test on the formatted output.